src/io/matgas.jl

```diff
@@ -292,9 +292,9 @@ function parse_m_string(data_string::String)
         case["source_version"] = "0.0.0+"
     end
 
-    required_metadata_names = ["mgc.gas_specific_gravity", "mgc.specific_heat_capacity_ratio", "mgc.temperature", "mgc.compressibility_factor", "mgc.units"]
+    required_metadata_names = ["mgc.gas_specific_gravity", "mgc.gas_molar_mass", "mgc.specific_heat_capacity_ratio", "mgc.temperature", "mgc.compressibility_factor", "mgc.units"]
```

Comment:
Why is gas_molar_mass both in required and optional meta data names.

Reply:
Sorry, this was a mistake. Should be fixed, now.